openhtf/io/output/mfg_inspector.py

```diff
@@ -129,14 +128,13 @@ def _EnsureUniqueParameterName(name, used_parameter_names):
 
 def _AttachJson(record, testrun):
   """Attach a copy of the JSON-ified record as an info parameter.
-
   Save a copy of the JSON-ified record in an attachment so we can access
   un-mangled fields later if we want.  Remove attachments since those get
   copied over and can potentially be quite large.
   """
   record_dict = data.ConvertToBaseTypes(record, ignore_keys=('attachments',))
   record_json = json_factory.OutputToJSON(
-      sort_keys=True, indent=2).encode(record_dict)
```

Comment:
When I enter that code block I get an error, unsure yet if it's a bug..investigating

  as_dict = data.ConvertToBaseTypes(test_record, ignore_keys='attachments')
  File "/home/source/openhtf/openhtf/util/data.py", line 113, in ConvertToBaseTypes
    assert not isinstance(ignore_keys, basestring), 'Pass a real iterable!'
AssertionError: Pass a real iterable!